pkg/controller/networkpolicy/networkpolicy_controller_test.go

```diff
@@ -2632,7 +2633,7 @@ func TestGetAppliedToWorkloads(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			actualPods, actualEEs, actualErr := c.getAppliedToWorkloads(tt.inATG)
+			actualPods, actualEEs, _, actualErr := c.getAppliedToWorkloads(tt.inATG)
```

Comment:
Please add tests to cover the Node NetworkPolicy case, otherwise it's hard to ensure the code is always correct.